docker/entry.sh

```diff
@@ -227,13 +248,7 @@ modprobe_deps() {
 ## Allow "exit 0", so that when used as an initContainers in Kubernetes, 
 ## next initContainers with a different distro will be tried
 
-host_dist="$(lbdisttool.py -l --os-release $HOSTRELEASE | awk -F'.' '/^rhel/ {print $1}' )"
-image_dist="$(lbdisttool.py -l | awk -F'.' '/^rhel/ {print $1}' )"
-
-if [[ -z $host_dist ]] || [[ $host_dist != $image_dist ]]; then
-	echo " The distro of the host does not match that of the image"
-	[[ $LB_SKIP == yes ]] && exit 0 || exit 1
-fi
+host_dist_matches_image_dist && [[ $LB_SKIP == yes ]] && exit 0 || exit 1
```

Comment:
I don't want it to exit if this isn't *explicitly* enabled. So 

```
if [[ $LB_SKIP == yes ]]; then
  your code
fi
```

Reply:
Also set `host_dist_matches_image_dist`  not to run when $LB_SKIP is not set

```bash
if [[ $LB_SKIP == yes ]]; then
	host_dist_matches_image_dist || exit 0 
elif [[ $LB_SKIP == no ]]; then
	host_dist_matches_image_dist || exit 1
fi
```